manifests/a/ApacheFriends/Xampp/8.1.1-2/ApacheFriends.Xampp.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ApacheFriends.Xampp
+PackageVersion: 8.1.1-2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --mode unattended --launchapps 0
+  SilentWithProgress: --mode unattended --launchapps 0
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/8.1.1/xampp-windows-x64-8.1.1-2-VS16-installer.exe/download
```

Comment:
This is not the download link I see at https://www.apachefriends.org/index.html

Reply:
Interesting, checked a older version and it does appear to be on the site, guess we'll use their server from now on